### DISCUSSION TITLE
Welcome to 30DaysOfJavaScript Discussions!

### DISCUSSION THREAD

**Discussion by swapnilsparsh** [swapnilsparsh/30DaysOfJavaScript#68] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tridib11** [comment]
Hey i dont know anything about javascript but i know abut java can i be a part of this?

**Reply by swapnilsparsh** [reply]
Sorry but this repo consists of JavaScript only.

**Reply by Tridib11** [reply]
Ok no issues

**Comment by HarshalMjn** [comment]
fix footer of News

**Reply by swapnilsparsh** [reply]
@HarshalMjn Thank You for reporting, will surely fix this.

**Comment by AYUSHMAN777** [comment]
![Screenshot (265)](https://github.com/swapnilsparsh/30DaysOfJavaScript/assets/110350591/7c3b24eb-9c5d-4cfa-b2ac-30c0403e9537)
Sir, I can improve this UI and it seems there is a padding issue too.
